pkg/services/ngalert/api/tooling/definitions/alertmanager_validation_test.go

```diff
@@ -255,3 +257,111 @@ func TestValidateRoutes(t *testing.T) {
 		}
 	})
 }
+
+func TestValidateMuteTimeInterval(t *testing.T) {
```

Comment:
Only light testing here. Making sure we are properly running the relevant code from the prometheus project. The goal isn't to deeply test Prometheus's validation.